### DISCUSSION TITLE
Kosovo Flag in Analytics Renders as Question Mark

### DISCUSSION THREAD

**Discussion by versecafe** [vercel/community#6350] [Bug Report]
### Page to Investigate

https://vercel.com/[team]/[project]/analytics

### Steps to Reproduce

<img width="1292" alt="image" src="https://github.com/vercel/community/assets/147033096/7ba65f73-2098-4375-91e7-74bbb07b60e8">

Kosovo flag is completely never sent, seems to be the only flag with any issue.

**Comment by amyegan** [comment]
Thanks for reporting this! I let the team know so we can get it fixed 🙇

**Comment by feugy** [comment]
Thanks a million for the report @versecafe, with all our apologies.
<img width="899" alt="image" src="https://github.com/vercel/community/assets/186268/185658c6-35cf-4d6a-bd2b-bdf9fde66c88">

This should be fixed now. Please re-open this issue if anything isn't as expected.
